doc/builders/fetchers.chapter.md

```diff
@@ -132,19 +132,24 @@ A number of fetcher functions wrap part of `fetchurl` and `fetchzip`. They are m
 
 `fetchFromGitHub` expects four arguments. `owner` is a string corresponding to the GitHub user or organization that controls this repository. `repo` corresponds to the name of the software repository. These are located at the top of every GitHub HTML page as `owner`/`repo`. `rev` corresponds to the Git commit hash or tag (e.g `v1.0`) that will be downloaded from Git. Finally, `hash` corresponds to the hash of the extracted directory. Again, other hash algorithms are also available, but `hash` is currently preferred.
 
+To use a different GitHub instance, use `githubBase` (defaults to `"github.com"`).
```

Comment:
It would be so much better to have a list here instead, as that would allow checking at a glance what's involved. But it's not your fault and I don't want to blow up the scope of your PR. Would be great if you took any of those functions and reformatted the argument descriptions into a list (or definition list, which gives slightly more semantics).

Ideally we'd get that documentation [immediately from the code](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchgithub/default.nix), but it's quite a stretch at the moment.